build/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.5.1-dev
+
+- Expose a set of valid inputs in `InvalidInputException`.
+- Build implementations are now expected to throw `InvalidInputException`s, the
+  check has been removed from this package.
```

Comment:
I don't understand this. Can we rephrase or expand it?

Reply:
The `BuildStepImpl` used to have a check that banned reads for all files outside of `lib` from the non-root package. It no longer has enough information to do this check itself (and no longer knows the root package either).

The underlying asset reader that it is given must implement this now.

Reply:
I adapted the changelog to point out that `AssetReader` implementations are supposed to perform the check now, and that it has been removed from the `BuildStepImpl`.